docs/installation.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+---
+
+# Installation
+
+TigerBeetle is a single small statically linked binary, download it here:
+
+|               | Linux                            | Windows                          | MacOs                             |
+| :------------ | :------------------------------- | :------------------------------- | :-------------------------------- |
+| x86_64        |[tigerbeetle-x86_64-linux.zip]    |[tigerbeetle-x86_64-windows.zip]  | [tigerbeetle-universal-macos.zip] |
+| aarch64       |[tigerbeetle-aarch64-linux.zip]   |             N/A                  | [tigerbeetle-universal-macos.zip] |
+
+[tigerbeetle-aarch64-linux.zip]: https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-aarch64-linux.zip
+[tigerbeetle-universal-macos.zip]: https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-universal-macos.zip
+[tigerbeetle-x86_64-linux.zip]: https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-x86_64-linux.zip
+[tigerbeetle-x86_64-windows.zip]: https://github.com/tigerbeetle/tigerbeetle/releases/latest/download/tigerbeetle-x86_64-windows.zip
+
+While not required, a Docker container is also available:
```

Comment:
```suggestion
While not required (TigerBeetle is designed to be easy to deploy as a single binary), a Docker container is also available:
```

Reply:
Adding a little redundancy here on purpose to communicate the "single binary" experience idea and make it obvious.

Reply:
:100: